include/swift/SIL/SILValue.h

```diff
@@ -582,14 +597,6 @@ inline llvm::raw_ostream &operator<<(llvm::raw_ostream &os,
 // Out of line to work around lack of forward declaration for operator <<.
 inline UseLifetimeConstraint
 OperandOwnershipKindMap::getLifetimeConstraint(ValueOwnershipKind kind) const {
-#ifndef NDEBUG
-  if (!canAcceptKind(kind)) {
-    llvm::errs() << "Can not lookup lifetime constraint: " << kind
-                 << ". Not in map!\n"
```

Comment:
I think this is needed here. Here is my suggestion, why don't we add two entry points, one that asserts and the other that returns an optional. That way the user can make the choice.

Reply:
Right, I reverted to original